src/sentry/debug_files/artifact_bundles.py

```diff
@@ -234,3 +242,54 @@ def _index_urls_in_bundle(
     ArtifactBundle.objects.filter(id=artifact_bundle.id).update(
         indexing_state=ArtifactBundleIndexingState.WAS_INDEXED.value
     )
+
+
+def maybe_renew_artifact_bundles(used_artifact_bundles: Dict[int, datetime]):
+    if options.get("sourcemaps.artifact-bundles.enable-renewal") == 1.0:
```

Comment:
Is this a float being used as a boolean?

Reply:
It's my bad, I did use it as float initially since I saw many people using floats but then I found it also supports bools